core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -4649,4 +4650,22 @@ private void addUniqueKeyHandler(MockTable table) {
           });
     }
   }
+
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6594">[CALCITE-6594]
+   * RelMdSize does not handle ARRAY constructor calls</a>. */
+  @Test void testSizeArrayConstructor() {
```

Comment:
I suggest moving this unit test to the middle of the test file. Place it in the same position as the other testing methods.